### PR TITLE
adds .data, fixes --checkdir

### PR DESCRIPTION
**Pull request by dmalan**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -60,5 +60,5 @@ class CustomInstall(install):
         "console_scripts": ["check50=check50:main"]
     },
     url="https://github.com/cs50/check50",
-    version="2.0.6"
+    version="2.0.8"
```
Comment (kzidane) [inline setup.py]: Why `8`?

Comment (dmalan) [reply setup.py]: Actually, let's wait before merging, couple other changes pending it seems. And we should go to 2.1.0 anyway.

Comment (kzidane) [reply setup.py]: :+1:

**Comment by dmalan** [comment]
Okay, ready for you, @kzidane!

File: check50.py [hunk 2]
```diff
@@ -67,6 +67,8 @@ def main():
                         help="display the full tracebacks of any errors")
 
     config.args = parser.parse_args()
+    if isinstance(config.args.checkdir, list):
```
Comment (crossroads1112) [inline check50.py]: Since the `nargs=1` line was removed, how could this ever be a list?

**Comment by crossroads1112** [comment]
This needs to be merged into `submit50` first: https://github.com/cs50/submit50/pull/147

**Pull request merged**